app/assets/stylesheets/application.scss

```diff
@@ -10,6 +10,6 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
- *= require_self
  */
+// Custom bootstrap variables must be set or imported *before* bootstrap.
+@import "bootstrap";
```

Comment:
Please add a blank line at the end of the file.

In the future follow this rule of keeping a blank line in a file.
Thanks.